docs/manual.rst

```diff
@@ -198,6 +198,11 @@ still be running, but it will disconnect the majority of peers (since connection
 to peers that already have the metadata are redundant). It will keep seeding the
 *metadata* only.
 
+Note that this doesn't prevent files from being created, which may happen
+especially if the torrent contains empty files. If you need to prevent that, you can either
```

Comment:
I think you can make this sentence more succinct and precise. Something like "This does not prevent empty files from being created, if the torrent contains any"